src/utils/priceTotal.ts

```diff
@@ -0,0 +1,8 @@
+import { CartItem } from "../domain/cart/types";
+
+export const totalPrice = (cart: CartItem[]) => {
+  return cart.reduce(
+    (accumulator, currentValue) => (accumulator += currentValue.price),
+    0
+  );
+};
```

Comment:
Since this fn is specific to the cart, you should move it to the domains/cart directory

Reply:
Done